frontend/src/store/modules/place/actions.js

```diff
@@ -40,9 +40,9 @@ export default {
         });
     },
 
-    fetchPlaces: (context) => {
+    fetchPlaces: (context, filters = {category: '', location: '', page: 1}) => {
         return new Promise((resolve, reject) => {
-            httpService.get('/places')
+            httpService.get('/places/search?filter[category]=' + filters.category + '&filter[location]=' + filters.location + '&page=' + filters.page)
```

Comment:
Better to create own method like `createSearchQueryUrl(filters)` which will return `/places/searc?filter[category]=' + filte...`